core/src/main/java/io/questdb/cairo/wal/ApplyWal2TableJob.java

```diff
@@ -574,6 +606,28 @@ private int processWalCommit(
                 writer.setSeqTxn(seqTxn);
                 writer.markSeqTxnCommitted(seqTxn);
                 lastCommittedRows = 0;
+
+                try (WalEventReader eventReader = walEventReader) {
```

Comment:
Would `MatVewRefresh` job add details like `LastRefreshBaseTableTxn` to the TRUNCATE transaction? I would expect only data commits to have this.